Jenkinsfile

```diff
@@ -35,7 +35,7 @@ metadata:
 spec:
   containers:
   - name: ${containerName}
-    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:${env.CONTAINER_VERSION}
+    image: gcr.io/pso-helmsman-cicd/jenkins-k8s-node:jenkins-cicd_images_builder-185
```

Comment:
Can we crest another global variable in Jenkins? I don’t want to redo all these PRs again